backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/database/repos/AcceptingStoresRepository.kt

```diff
@@ -118,35 +111,12 @@ object AcceptingStoresRepository {
             }.firstOrNull()?.let { it[AcceptingStores.id].value }
     }
 
-    fun importAcceptingStores(stores: List<CSVAcceptingStore>, project: ProjectEntity, dryRun: Boolean): StoreImportReturnResultModel {
```

Comment:
I refactored and moved this function to the service layer because I think conceptually the repository should only handle database operations (CRUD).
Validation and building a response are more like business logic.
But feel free to object.